### DISCUSSION TITLE
How can I change the public address of any station from source code or administrative configuration?

### DISCUSSION THREAD

**Discussion by draven3166** [AzuraCast/AzuraCast#7509] [Ideas]
When creating a radio instance, azuracast generates by default a url like this: https://my-domain/public/station-test
My question is how can I change from that public url to one that I can configure, for example:
https://my-domain/name-client/station-test
To improve work with clients, it is also possible to create a proxy with redirection to the default public url generated by azuracast. (common solution)
Please if you could customize this by code and/or if there is the possibility from the administrative panel, this in a generic way for all the radio instances that you can create.
Great work azuracast community, gretting regards.

**Comment by Heimatradio** [comment]
Why don't you create subdomains for your clients? That would be the easiest way.
https://client-name-1.my-domain
https://client-name-2.my-domain
etc.

**Reply by draven3166** [reply]
Thanks for the comment, _I have considered that optimal solution._
But I would like to be able to join the Azuracast development community to learn and contribute, thank you!

**Comment by BusterNeece** [comment]
Currently, we don't support fully custom URLs for station listen paths, as there's a possibility of creating a URL that collides with one of our own internal routes (either currently or in the future).

You can vote for it as a feature request at https://features.azuracast.com/, but be aware that there are some technical obstacles that prevent it from being a likely reality in the near future.
